### ISSUE TITLE
Discussion: Which "news" sources to include

### ISSUE THREAD

**Issue by hexonaut** [peerfact/peerfact.github.io#1]
The low hanging fruit of obviously fake news sites is an easy first thing to add. It is probably worthwhile to consider adding misleading, click-bait and possibly satire articles as well with different colors.

**Comment by warpraptor0** [comment]
I think one thing that will be crucial is making sure that the list doesn't get skewed too far in one direction or another.

Been thinking about doing something very similar that would tag shares based on the diversity of people sharing.

For example, it will be more difficult to flag articles based on content only but I think if you're sharing an article that has MOSTLY been shared by people who think like you think, then there's a higher probability that it is not straight news, but biased or more editorial than news.

**Comment by hexonaut** [comment]
Yes I agree about the political bias skewing, and I want to avoid that. Using the people who shared the article as a data source is an interesting idea. It seems some sort of algorithm is needed for evaluating sources with misleading/click-bait titles. I could see this being used as a factor in that determination.

**Comment by Paulooze** [comment]
I don't want to start a new issue for my question, because it fits the headline of this one. Are you considering adding localized (non-English) sources as well? We have a database of fake/untrustworthy websites (compiled by professionals). Here is the link (in Slovak): https://www.konspiratori.sk/.

**Comment by hexonaut** [comment]
I don't see why not. The plugin is in English right now, but there isn't much text to change. I've added the extension just now to here: https://github.com/peerfact/extension

Feel free to submit a PR.

**Comment by hexonaut** [comment]
Updated the title to be more specific. :)

**Comment by fpruitt** [comment]
So, is the view here that if a site shares something 'fake' once, that they are potentially blacklisted from that point forwards (if they are reported)? Or, does there have to be a frequency or pattern to it?

Seems like, especially with smaller sites, it would be easy to accidentally get blacklisted.

**Comment by alanmun** [comment]
Hi guys I'm on mobile and new to github at the same time, so here is my contribution. It looks like you missed: denverguardian.com

Searching the name of the site on Google gives you an article explaining how the site is fake news

**Comment by alanmun** [comment]
I didn't see it in your Google docs so if you add it its pretty much definitely a category 1

**Comment by hexonaut** [comment]
@fpruitt This is just a first step. Realistically I think we need to move from an all or nothing approach to some sort of quality rating. Small sites that slip up once or twice are not the focus of this plugin.

@DamaDamaDama Thanks for reporting; will add that in a moment.
